joomla/joomla-cms
# Add layout option to com_contact category menu item

Added `contact_layout` field to the com_contact category view.

This allows users to select the layout for the contact view that is for the com_contact category menu item.

Pull Request for Issue # .

### Summary of Changes

I added a single field to the com_contact `category` view. It allows users to select the layout that is used for the `contact` view. 

### Testing Instructions

1. Setup new Joomla installation.
1. Create template override for com_contact `contact` view.
1. Create a contact category.
1. Create a contact in the new category.
1. Create a contact category menu item.
1. Note missing field.

Verify the change...
1. Implement the code from this PR.
1. Note the field is not missing anymore.
1. Select the template override created in step 1, and save the menu item.
1. Note override being used on the front-end contact view for the menu item.

### Expected result

There should be a "Choose a Layout" field in the "Contact Display Options" tab.

### Actual result

The `presentation_style` field is present but not the "Choose a Layout" field.

This result can be achieved by adding the field to the category view .xml file. 

### Documentation Changes Required

I could not find any documentation on the com_contact category menu item.
